### DISCUSSION TITLE
Slides das Palestras de 2021 a 2022

### DISCUSSION THREAD

**Discussion by gabubellon** [apyb/comunidade#104] [Geral]
Olá,
Gostaria de fazer uma chamada para toda a comunidade em ajudar a atualizar o repositório de palestras dos eventos de 2020, 2021 e 2022 (e algum outro que ainda não esteja completo)

Convido a @apyb/diretoria-2020-2021  e @apyb/diretoria  assim como todos os organizadores dos eventos (EU que vos escrevo i inclusive) a fazer uma chamada publica a isso 
Podiamos aproveitas as novas redes sociais de Apyb e posta lá né

O repositório:
https://github.com/pythonbrasil/talks

Se você está lendo isso e participou como palestrante ou keynote em um dos eventos, envie o seu slide pra lá. Caso não queira/tenha tempo/não saiba como adicionar, coloca o mesmo em um link como comentário aqui que eu adiciono. 

Bora deixar ainda mais completo os conteúdo dos eventos!!

**Comment by naanadr** [comment]
Muito bem lembrado @gabubellon , já abri um PR no repositório e aproveitei para abrir umas issues. 

Acho que podemos sim utilizar as redes sociais da APyB para divulgar o repositório, vamos alinhar internamente e retornar depois com mais informações.

**Reply by gabubellon** [reply]
@naanadr Obrigado pela ajuda

Lembrando que a responsabilidade disso cabe a cada organizador e organização do evento, mas seria sim importante ter isso pelo menos mapeado
